setup.py

```diff
@@ -53,8 +53,12 @@ class BuildProtoCommand(Command):
   def initialize_options(self):
     self.skip_proto = False
     try:
-      prefix = subprocess.getoutput(
-          'pkg-config --variable prefix protobuf').strip()
+      if os.name == 'posix' and sys.version_info[0] < 3:
```

Comment:
Also handled by future. In general, try future before switching on sys.version_info for 3 => 2 compatibility, that's what it's for and it's guaranteed to keep working as Python3 evolves.

http://python-future.org/compatible_idioms.html#commands-subprocess-modules